### PR TITLE
add summaries endpoint

### PR DESCRIPTION
**Pull request by chaig15**

### CODE REVIEW

File: polygon/rest/models/summaries.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+from sqlite3 import Timestamp
+from typing import Optional
+from ...modelclass import modelclass
+from .tickers import Branding
+
+
+@modelclass
+class Session:
+    "Contains Session data for the summaries endpoint."
+    change: Optional[float] = None
```
[the remark this answers is not in the document]
Comment (chaig15) [reply polygon/rest/models/summaries.py:10]: few of these seem optional in the mock response, if you see early trading change, percent and late_trading_change and percent.

File: polygon/rest/models/summaries.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    def from_dict(d):
+        return Options(**d)
+
+
+@modelclass
+class SummaryResult:
+    "Contains summary result data for a list of tickers"
+    price: Optional[float] = None
+    name: Optional[str] = None
+    ticker: Optional[str] = None
```
[the remark this answers is not in the document]
Comment (chaig15) [reply polygon/rest/models/summaries.py:48]: in the case of an error, as seen in the APX response. We only get back ticker, error, and message though right?

File: polygon/rest/models/summaries.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    branding: Optional[Branding] = None
+    market_status: Optional[str] = None
+    type: Optional[str] = None
+    session: Optional[Session] = None
```
[the remark this answers is not in the document]
Comment (chaig15) [reply polygon/rest/models/summaries.py:52]: in the case of an error, as seen in the APX response. We only get back ticker, error, and message though right?

File: polygon/rest/models/summaries.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jbonzo) [reply polygon/rest/models/summaries.py:48]: You are correct. Good catch

**Pull request merged**